static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -119,6 +119,13 @@ function SetupDocs({organization, projects, search}: Props) {
       project_id: newProjectId,
     });
     browserHistory.push(`${window.location.pathname}?${searchParams}`);
+    if (clientState) {
```

Comment:
@Neo-Zhixing I'm confused why we wouldn't update client state if it's null at this point

Reply:
because we don't know the other fields in clientState yet - this is more like a PUT than a POST in some sense.